ci_container.sh

```diff
@@ -0,0 +1,8 @@
+IMAGE_NAME=wptd-testrun-jenkins
+
+docker build -t $IMAGE_NAME .
```

Comment:
Can we incorporate a version number here?

Reply:
The image we push for the Jenkins bots to pull needs to have a constant name (otherwise updating the image will require changing the version in Jenkins).

Reply:
I think the standard practice for this is to version images, and then use a `:latest` version/tag for use cases such as that. We could also use a version name like `:jenkins` if we want.